source/index.html.md

```diff
@@ -787,6 +787,59 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
+
+##### Method: `googlepay`
+
+To accept a payment using Google Pay, the complete payment token, recipient ID
+and derived shared secret, are required. Please refer to the [official
+documentation][GooglePay-PaymentCryptography]. Only protocol version `ECv2` is
+supported.
+
+<dl class="dl-vertical">
+  <dt>googlepay[token]
+    <span class="type">[:json:]</span>
+  </dt>
+  <dd>
+    Raw payment method token as received in response from Google. UTF-8 encoded
+    serialization of a JSON dictionary.
+  </dd>
+  <dt>googlepay[shared_key]
+    <span class="type">[:base64:]</span>
+  </dt>
+  <dd>
+    The shared secret derived from the ephemeral public key and your private
+    key.
+  </dd>
+  <dt>googlepay[recipient_id]
+    <span class="type">[\x21-\x7E]+
+      <a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">ASCII printable characters</a> excluding space
+    </span>
+  </dt>
+  <dd>
+    The Google Pay recipient of the payload, e.g. <code>merchant:0123456789</code>.
+  </dd>
+</dl>
+
+<p class="alert alert-info">
+  <b>Notice:</b> Signing is required to use the <code>googlepay</code> payment
+  method.
+  <br />
+  <b>Notice:</b> An authorization made with <code>googlepay</code> is strongly
+  authenticated (SCA in PSD2) if <code>authMethod</code> is
+  <code>CRYPTOGRAM_3DS</code> and the Google Pay guidelines for
+  <i><a href="https://developers.google.com/pay/api/web/guides/resources/sca">sca</a></i>
+  have been followed.
+
+  If <code>authMethod</code> is <code>PAN_ONLY</code>, a 3-D Secure flow is
+  required for SCA.
```

Comment:
```suggestion
  If <code>authMethod</code> is <code>PAN_ONLY</code>, a 3-D Secure flow is
  necessary to obtain strong authentication.
```

Dunno which version is easier to understand correctly 🤷‍♂️

Reply:
I'd prefer to use "SCA", as it's probably easily recognizable as SCA related (and briefer).
But I have no strong opinion. You're welcome to merge if you prefer your edition.